mk-files/docs.mk

```diff
@@ -68,10 +65,8 @@ release-docs:
 		git checkout master && \
 		git checkout -b $(STAGING_BRANCH) && \
 		$(call dry-run,git push -u origin $(STAGING_BRANCH)); \
-	fi && \
-	git checkout -B $(CURRENT_SHORT_MINOR_VERSION)-post origin/$(STAGING_BRANCH) && \
-	$(call dry-run,git push -u origin $(CURRENT_SHORT_MINOR_VERSION)-post) && \
-	if [[ $(CLEAN_VERSION) == *.0 ]]; then \
+		git checkout -B $(CURRENT_SHORT_MINOR_VERSION)-post origin/$(STAGING_BRANCH) && \
```

Comment:
Do we still need the capital "-B" here, or would lowercase "-b" work fine? (I've always been confused by this)

Reply:
Lowercase `-b` should work here. For non-patch releases (this case here), these branches don't exist yet anyway so there shouldn't be a difference between `-b/-B`.